### PR TITLE
Merge in networking support from ejs/net2 branch to master.

### PR DESCRIPTION
**Pull request by eswenson1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,5 +56,33 @@ from scratch.
...
+
+In order to change the IP address of the host, you must update multiple
+places and rebuild ITS and the host table. You will also have to modify
+dskdmp.ini.  Here are the places where the IP address is configured (and all
+of these must agree):
+
+dskdmp.ini (in build/klh10 directory on the host)
+SYSTEM; CONFIG > (ITS configuration file)
+SYSHST; H3TEXT > (host table source file)
+SYSBIN; HOSTS3 > (binary file compiled from SYSHST; H3TEXT >)
```
Comment (larsbrinkhoff) [inline README.md]: This formatting doesn't come out right when you view README.md in GitHub.  You need to add to spaces at the end of a line to force a line break.

Comment (eswenson1) [inline README.md]: Thx. I'll fix in next commit.

**Comment by larsbrinkhoff** [comment]
> I'm not sure what I was supposed to have done differently.

Instead of `git merge origin/master` (or other branch), you could have done `git rebase origin/master`.  Make sure to fetch the latest origin/master first.

I just tried this myself with your `ejs/net2` branch.  After resolving the conflicts, this is the result:

https://github.com/PDP-10/its/compare/lars/rebased-ejs/net2

Hmm, there's still one merge commit.

**Comment by eswenson1** [comment]
I *did* do a "git rebase origin/master" after previously having done a 
fetch. I ended up with my working tree on the master branch, and when I 
switched back to the ejs/net2 branch, it was unchanged.  And it still 
didn't have the DOC changes. So I merged with master and it worked. -- Eric

**Comment by larsbrinkhoff** [comment]
Is it ok if I modify this pull request?  I can tell you step by step what I do.

**Comment by larsbrinkhoff** [comment]
> And it still didn't have the DOC changes. So I merged with master and it worked.

That's strange.  I can't explain that.  But as far as I know, in normal circumstances, it should have worked.

**Comment by eswenson1** [comment]
Absolute. Feel free to modify the PR. Let me know what you did (for educational purposes).

**Comment by larsbrinkhoff** [comment]
Ok!  First of all, I do `git fetch -p` to update my repository and prune off branches that have been deleted upstream.  Then I do `git checkout master` and `git merge --fast-forward origin/master` to ensure that my `master` is up to date.

Then, `git checkout -t ejs/net2` to get a copy of your branch, and `git rebase master` to move the commits on top of `master`.  There are a few merge conflicts, but they are easy to resolve.

Now there are these five commits:

- Merged in lars/config-net branch.
- Added support for FTP client and server.
- Added SUPDUP client and server support.
- Added TELNET server and client.
- Integrated basic ITS networking support and host table.

**Comment by larsbrinkhoff** [comment]
Now, I want to fix the first commit where the formatting in README.md is off.  Also last commit just adds build/klh10/config.203, which I think should be merged into the first commit.  And the first commit still contains src/system/config.202 for some reason.

To fix these issues, I do `git rebase -i master`.  I have `GIT_EDITOR=emacsclient`, so Emacs pops up a buffer with this:

    pick 444e57f Integrated basic ITS networking support and host table.
    pick 3fd92c3 Added TELNET server and client.
    pick 6ba6048 Added SUPDUP client and server support.
    pick 41e42e4 Added support for FTP client and server.
    pick 3d88343 Merged in lars/config-net branch.

I edit this to:

    e 444e57f Integrated basic ITS networking support and host table.
    f 3d88343 Merged in lars/config-net branch.
    pick 3fd92c3 Added TELNET server and client.
    pick 6ba6048 Added SUPDUP client and server support.
    pick 41e42e4 Added support for FTP client and server.

These are actually commands to git for how to handle each commit.  `pick` means to add it unmodified.  `e` means to edit it.  `f` means to merge it into the commit above.

So in this case, I want to edit the first commit to fix the formatting and delete the config file in src.  And then the last commit with build/klh10/config.203 should be added to that.

I save the buffer and type C-x # to exit emacsclient and have git proceed with the rebase.

**Comment by larsbrinkhoff** [comment]
Now, git says:

    Stopped at 444e57fcef24923e9d244a1bdff267b68d6b981a... Integrated basic ITS networking support and host table.
    You can amend the commit now, with
        git commit --amend
    Once you are satisfied with your changes, run
        git rebase --continue

I type `git rm src/system/config.202`, edit README.md, and then `git add README.md`.  `git status -s` now says:

    M  README.md
    D  src/system/config.203

I add these changes to the commit by saying `git commit --amend -C HEAD`.  And then `git rebase --continue`.

**Comment by larsbrinkhoff** [comment]
Now the interactive rebase is finished.  We're left with these commits:

- Integrated basic ITS networking support and host table.
- Added TELNET server and client.
- Added SUPDUP client and server support.
- Added support for FTP client and server.

**Comment by larsbrinkhoff** [comment]
Now I scrutinize each commit in GitHub, and wait for Travis to report the build status.

**Comment by larsbrinkhoff** [comment]
Oops, the first commit adds a config.203, but doesn't remove the old config.202.  So I do a `git rebase -i master` again to clean that up.

**Comment by larsbrinkhoff** [comment]
Ok, I think we're done now.  Commits look good, Travis is happy, and I tested telnet and ftp against KLH10.

**Comment by larsbrinkhoff** [comment]
Now, if you want to continue using this branch, do a `git fetch`, `git checkout ejs/net2`, and `git reset --hard origin/ejs/net2`.  **WARNING**, this will completely overwrite anything you had before on that branch.

**Comment by eswenson1** [comment]
I did that on purpose. I wanted to have both the pre- and 
post-networking configs in there. But feel free to kill the old if you want.

**Pull request merged**